code/workspaces/client-api/src/dataaccess/stackUrlService.js

```diff
@@ -14,16 +14,16 @@ export default function notebookUrlService(notebook, user) {
   if (notebook.type === ZEPPELIN) {
     return requestZeppelinToken(notebook, user)
       .then(createZeppelinUrl(notebook));
-  } else if (notebook.type === JUPYTER) {
+  } if (notebook.type === JUPYTER) {
     return requestJupyterToken(notebook, user)
       .then(createJupyterUrl(notebook));
-  } else if (notebook.type === JUPYTERLAB) {
+  } if (notebook.type === JUPYTERLAB) {
     return requestJupyterToken(notebook, user)
       .then(createJupyterlabUrl(notebook));
-  } else if (notebook.type === RSTUDIO) {
+  } if (notebook.type === RSTUDIO) {
     return requestRStudioToken(notebook, user)
       .then(createRStudioUrl(notebook));
-  } else if (notebook.type === NBVIEWER) {
+  } if (notebook.type === NBVIEWER) {
```

Comment:
Out of interest, how come the `else`s have been removed?

Reply:
Change in linting rules. As the blocks have a return statement they terminate the function and therefore the `else` clause is not needed.